compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/CoreTextField.kt

```diff
@@ -301,13 +304,17 @@ internal fun CoreTextField(
         state.hasFocus = it.isFocused
 
         if (textInputService != null) {
-            notifyTextInputServiceOnFocusChange(
-                textInputService,
-                state,
-                value,
-                imeOptions,
-                offsetMapping
-            )
+            if (state.hasFocus && enabled && !readOnly) {
```

Comment:
It seems this is the only thing that we should take a look

Reply:
This change disables input methods. It shouldn't break it for desktop/ios. I checked desktop with enabled state - it still works the same.